liota/lib/utilities/utility.py

```diff
@@ -110,6 +111,28 @@ def get_linux_version():
     return platform.platform()
 
 
+def get_default_network_interface():
+    """
+    Works with Linux.
+    :return: Default Network Interface of the Edge_System
+    """
+    cmd = "route | grep '^default' | grep -o '[^ ]*$'"
```

Comment:
There are situations where route may not actually return a default route in the main routing table, as the default route might be kept in another table.  This isn't a 'normal' circumstance, but something to be aware of